vue-components/src/components/util/generateUid.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Generates an ID that can be used within components, e.g. for binding an input id to a label's `for` attribute
+ *
+ * @param {string} componentName
```

Comment:
Our eslint config forced me to add `@param` and `@return` with type annotations here. This doesn't make much sense to me since we're using TS. Thoughts on disabling `jsdoc/require-param` and `jsdoc/require-returns`?

Reply:
I didn't even know we had those enabled... I'm good either / or